packages/ag-charts-community/src/chart/series/cartesian/barUtil.ts

```diff
@@ -302,10 +304,11 @@ export function resetBarSelectionsFn(_node: Rect, { x, y, width, height, clipBBo
 }
 
 export function computeBarFocusBounds(
+    series: ISeries<any, any>,
```

Comment:
nit: `ISeries<any, any>` is a bloated and contagious type. In util functions, it's preferable to explicitly redeclare the specific properties required:
```suggestion
    series: { contentGroup: Group },
```

Reply:
Duplicating the interfaces like this makes more work if you ever change stuff like `contentGroup` - which we have done in the past

Reply:
Yes that's true. However, the use of `any` is definitely an anti-pattern and we mustn't encourage its use. As compromise, you could try to see whether one of the alternatives is sufficient:
* `ISeries<never, never>`
* `ISeries<unknown, unknown>`
* `ISeries<{}, {}>`

Reply:
Updated